README.md

```diff
@@ -1512,10 +1514,19 @@ You can examine the value of `Sunspot::Rails.configuration` at runtime.
 
 ## Running Solr in production environment
 
-`sunspot_solr` gem is an easy and convenient way to start your development with Solr.
-However once you are ready to deploy your code to a production, consider using another options like
-[standalone](https://lucene.apache.org/solr/guide/installing-solr.html) or
-[docker](https://hub.docker.com/_/solr/) Solr setup
+`sunspot_solr` gem is a convenient way to start working with Solr in development.
+However, it is not suitable for production use. Below are some options for deploying Solr:
+
+1. [Standalone](https://lucene.apache.org/solr/guide/installing-solr.html) or
+2. [Docker](https://hub.docker.com/_/solr/) Solr setup (also a good alternative for development)
+3. [Chef](https://supermarket.chef.io/cookbooks/solr_6/versions/0.2.0) (can be used with solr 7 as well)
+4. [Ansible](https://github.com/geerlingguy/ansible-role-solr)
+5. [Kubernetes](https://hub.helm.sh/charts/incubator/solr) This deploys a Zookeeper cluster so you will need to convert cores
+   to collections in order to use it.
+
+You can also use Docker Solr for development which, regardless of how you deploy in production, will let you match
```

Comment:
Love this description!
Do you think we can provide a sample docker run command here to show how easily one could be used?

Reply:
Yup, I'll add that.